package.json

```diff
@@ -26,14 +26,15 @@
     "clsx": "^2.1.0",
     "geist": "^1.2.2",
     "immer": "^10.1.1",
-    "lucide-react": "^0.312.0",
+    "lucide-react": "^0.447.0",
```

Comment:
Why is this being updated?

Reply:
In an effort to try and create continuity with our website, I was trying to use lucide-react icons for the social media icons in the team-members section.

They are deprecating a lot of brand icons, including GitHub. Their suggested alternative was doing the same thing.

I uninstalled this before realizing that I did not install it originally, so I re-installed it, but did not get the version we were on previously.

Reply:
This is fine. Keep it.